### PR TITLE
Update windows-update-troubleshooting.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #4151 
remove the broken link

### CODE REVIEW

**Comment by jvsam** [comment]
Thank you @MaratMussabekov .

@nenonix and @JohanFreelancer9 for copy review please. Thank you.

File: windows/deployment/update/windows-update-troubleshooting.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Or
 DownloadManager [0]12F4.1FE8::09/29/2017-13:45:08.530 [agent]DO job {C6E2F6DC-5B78-4608-B6F1-0678C23614BD} hit a transient error, updateId = 5537BD35-BB74-40B2-A8C3-B696D3C97CBA.201 <NULL>, error = 0x80D0000A 
 ``` 
  
-Go to Services.msc and ensure that Windows Firewall Service is enabled. Stopping the service associated with Windows Firewall with Advanced Security is not supported by Microsoft. For more information , see [I need to disable Windows Firewall](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc766337\(v=ws.10\)) or [Windows Update stuck at 0 percent on Windows 10 or Windows Server 2016](https://support.microsoft.com/help/4039473/windows-update-stuck-at-0-percent-on-windows-10-and-windows-server-201).
+Go to Services.msc and ensure that Windows Firewall Service is enabled. Stopping the service associated with Windows Firewall with Advanced Security is not supported by Microsoft. For more information , see [I need to disable Windows Firewall](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc766337\(v=ws.10\)).
```
Comment (JohanFreelancer9) [inline windows/deployment/update/windows-update-troubleshooting.md]:
```suggestion
Go to Services.msc and ensure that Windows Firewall Service is enabled. Stopping the service associated with Windows Firewall with Advanced Security is not supported by Microsoft. For more information, see [I need to disable Windows Firewall](https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc766337(v=ws.10)).

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
your recommendations were applied,
Thank you!

**Comment by jvsam** [comment]
Hello @greg-lindsay this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you. See: [commits](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4333/commits) and [changes](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4333/files).

**Comment by jvsam** [comment]
Hi again @greg-lindsay if time permits, kindly review and approve this PR that contains removal of the broken link. Right now, users clicking the link are getting redirected to a 404 page. We appreciate the help, thank you.

CC: @AndreaBarr

**Comment by jvsam** [comment]
Hi again @greg-lindsay and @AndreaBarr I'd like to follow-up on the status of this pull request. Can you please review and merge if there are no issues? Thank you so much.

**Comment by jvsam** [comment]
Hi @greg-lindsay and @AndreaBarr just following up again. Kindly review this PR when time permits. We appreciate the help, thank you.

**Comment by jvsam** [comment]
Hi @greg-lindsay following up again on this open PR. We hope you can review this soon. @AndreaBarr can you please check as well? Please let us know if there are issues. Thank you again.

**Comment by jvsam** [comment]
Hi again @AndreaBarr would you be able to assist? This PR has been opened for 2 weeks and it contains a fix to a broken link (the target page of the outgoing link doesn't exist anymore). Thank you so much.

**Comment by jvsam** [comment]
Thank you once again @AndreaBarr!

**Pull request merged**